xml/deploy_cephadm.xml

```diff
@@ -757,6 +757,164 @@ o- time_server ................................................ [enabled]
+         <term>ms_mon_service_mode</term>
+         <listitem>
+           <para>
+             A list of permitted modes for clients or other &ceph; daemons
+             to use when connecting to monitors.
+           </para>
+         </listitem>
+       </varlistentry>
+       <varlistentry>
+         <term>ms_mon_client_mode</term>
+         <listitem>
+           <para>
+             A list of connection modes, in order of preference, for clients
+             or non-monitor daemons to use when connecting to monitors.
+           </para>
+         </listitem>
+       </varlistentry>
+     </variablelist>
+     <para>
+       In order to enable MSGR2 encryption mode during the deployment, you need
+       to add some configuration options to the &cephsalt; configuration before
+       running <command>ceph-salt apply</command>.
+     </para>
+     <para>
+       To use <literal>secure</literal> mode, run the following commands.
+     </para>
+     <para>
+       Add the global section to <filename>ceph_conf</filename> in the &cephsalt;
+       config tool:
+     </para>
+<screen>&prompt.smaster;ceph-salt config /cephadm_bootstrap/ceph_conf add global</screen>
+     <para>
+       Set the following options:
+     </para>
+<screen>&prompt.smaster;ceph-salt config /cephadm_bootstrap/ceph_conf/global set ms_cluster_mode "secure crc"
+&prompt.smaster;ceph-salt config /cephadm_bootstrap/ceph_conf/global set ms_service_mode "secure crc"
+&prompt.smaster;ceph-salt config /cephadm_bootstrap/ceph_conf/global set ms_client_mode "secure crc"
+</screen>
+     <note>
+       <para>
+         Ensure <literal>secure</literal> preceeds <literal>crc</literal>.
+       </para>
+     </note>
+     <para>
+       To <emphasis>force</emphasis> <literal>secure</literal> mode, run the following commands:
+     </para>
+<screen>&prompt.smaster;ceph-salt config /cephadm_bootstrap/ceph_conf/global set ms_cluster_mode secure
+&prompt.smaster;ceph-salt config /cephadm_bootstrap/ceph_conf/global set ms_service_mode secure
+&prompt.smaster;ceph-salt config /cephadm_bootstrap/ceph_conf/global set ms_client_mode secure
+</screen>
+     <tip xml:id="update-inflight-encryption-settings">
+       <title>Updating Settings</title>
+       <para>
+         If you want to change any of the above settings, set
+         the configuration changes in the monitor configuration store. This is
+         achieved using the <command>ceph config set</command> command.
+       </para>
+<screen>&prompt.smaster;ceph config set global <replaceable>CONNECTION_OPTION</replaceable> <replaceable>CONNECTION_MODE</replaceable> [--force]</screen>
+     </tip>
+     <para>
+       For example:
+     </para>
+<screen>&prompt.smaster;ceph config set global ms_cluster_mode "secure crc"</screen>
+     <para>
+       If you want to check the current value, including default value, run
+       the following command:
+     </para>
+<screen>&prompt.smaster;ceph config get osd ms_cluster_mode</screen>
```

Comment:
Maybe we need to be more specific with: 
<screen>&prompt.smaster;ceph config get <replaceable>CEPH_COMPONENT</replaceable> <replaceable>CONNECTION_OPTION</replaceable></screen>

For example, to get the `ms_cluster_mode` for osd's run:
<screen>&prompt.smaster;ceph config set global <replaceable>CONNECTION_OPTION</replaceable> <replaceable>CONNECTION_MODE</replaceable> [--force]</screen>